cluster-service/deploy/helm/templates/deployment.yaml

```
@@ -0,0 +1,215 @@
+      initContainers:
+      - name: init
+        image: '{{ .Values.imageRegistry }}/{{ .Values.imageRepository }}:{{ .Values.imageTag }}'
+        imagePullPolicy: IfNotPresent
+        volumeMounts:
+        - name: rds
+          mountPath: /secrets/rds
+        - name: service
+          mountPath: /secrets/service
+        command:
+        - /usr/local/bin/clusters-service
+        - init
+        - --db-host=@/secrets/rds/db.host
+        - --db-port=@/secrets/rds/db.port
+        - --db-name=@/secrets/rds/db.name
+        - --db-user=@/secrets/rds/db.user
+        - --db-password=@/secrets/rds/db.password
+        - --db-disable-tls={{ .Values.databaseDisableTls }}
+        - --db-auth-method={{ .Values.databaseAuthMethod }}
+        - --force-migration={{ .Values.forceMigration }}
+        - --batch-processes-dry-run={{ .Values.batchProcessesDryRun }}
+        - --batch-processes={{ .Values.batchProcesses }}
+      containers:
+      - name: service
+        image: '{{ .Values.imageRegistry }}/{{ .Values.imageRepository }}:{{ .Values.imageTag }}'
+        imagePullPolicy: IfNotPresent
+        volumeMounts:
+        - name: service
+          mountPath: /secrets/service
+        - name: shards
+          mountPath: /secrets/shards
+        - name: rds
+          mountPath: /secrets/rds
+        - name: authentication
+          mountPath: /configs/authentication
+        - name: region-constraints
+          mountPath: /configs/region-constraints
+        - name: proxy
+          mountPath: /configs/proxy
+        - name: mixin-pull-secret
+          mountPath: /secrets/mixin-pull-secret
+        - name: instance-types
+          mountPath: /configs/cloud-resources/instance-types.yaml
+          subPath: instance-types.yaml
+        - name: instance-type-constraints
+          mountPath: /configs/cloud-resource-constraints/instance-type-constraints.yaml
+          subPath: instance-type-constraints.yaml
+        - name: cloud-regions
+          mountPath: /configs/cloud-resources/cloud-regions.yaml
+          subPath: cloud-regions.yaml
+        - name: cloud-region-constraints
+          mountPath: /configs/cloud-resource-constraints/cloud-region-constraints.yaml
+          subPath: cloud-region-constraints.yaml
+        - name: azure-credentials
+          mountPath: /secrets/azure-credentials
+        - name: keyvault
+          mountPath: "/secrets/keyvault"
+          readOnly: true
+        - name: azure-runtime-config
+          mountPath: /configs/azure-runtime-config
+        - name: azure-operators-managed-identities-config
+          mountPath: /configs/azure-operators-managed-identities-config.yaml
+          subPath: azure-operators-managed-identities-config.yaml
+        command:
+        - /usr/local/bin/clusters-service
+        - serve
+        - --log-level={{ .Values.logLevel }}
+        - --namespace={{ .Release.Namespace }}
+        - --runtime-mode={{ .Values.runtimeMode }}
```

We could hardcode the mode here as we are in the context of aro-hcp.
```suggestion
        - --runtime-mode=aro-hcp
```